.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: An actionable title e.g. "Update the icon for..." or "Fix the behavior of..."
```

Comment:
I think this is what will appear in the title section of the issue as a placeholder.

Reply:
I think for bugs, the actionable thing is always going to be "Fix the behavior of..." What if, instead, we request a summary of the unexpected behavior, e.g., "Uploading board member headshot results in an error"?